docs/compressing-images.md

```diff
@@ -0,0 +1,46 @@
+# TEAMMATES Compressing Images
+
+This document describes the steps to compress an image to be used in the TEAMMATES website.
+
+Two choices are provided, (i) online service (ii) command-line. Command-line is documented should the need of compressing large amount of images arise.
+
```

Comment:
I wouldn't mind a bit more informal tone :-p. e.g. `Use the command line if there are many images to compress.`